internal/examples/proto/src/einride/bigquery/public/v1/dogecoin_transaction.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+package einride.bigquery.public.v1;
+
+option go_package = "go.einride.tech/internal/examples/proto/gen/einride/bigquery/public/v1;publicv1";
+
+import "google/protobuf/timestamp.proto";
+import "google/type/date.proto";
+
+// Protobuf schema for the BigQuery public table:
+//
+//  bigquery-public-data.crypto_dogecoin.transactions
+message DogecoinTransaction {
+  string hash = 1; // STRING REQUIRED
```

Comment:
If this is supposed to be required in the BQ schema, how is that captured in the proto definition?

Reply:
It's not - everything is optional in the protos.